### DISCUSSION TITLE
Tabs组件RTL示例选择fixed样式有问题

### DISCUSSION THREAD

**Discussion by user-spider** [ant-design/ant-design-mobile#6685] [Q&A]
Tabs组件RTL示例选择fixed，active横线样式有问题，再点击其他tab样式会恢复正常，只有初始选择fixed有问题

**Comment by Layouwen** [comment]
> Tabs组件RTL示例选择fixed，active横线样式有问题，再点击其他tab样式会恢复正常，只有初始选择fixed有问题

方便在比如 codesandbox 之类的在线平台, 提供复现 demo 吗

**Reply by user-spider** [reply]
你好，在咱们官网示例就可以看到效果的，直接打开官网Tabs示例，点RTL示例的fixed，可以看到active小线条挤到最右边了

**Reply by user-spider** [reply]
https://ant-design-mobile.antgroup.com/zh/components/tabs

**Reply by Layouwen** [reply]
> https://ant-design-mobile.antgroup.com/zh/components/tabs

已经修复了，后面会更新
